### PR TITLE
Remove "deeply read article footer" AB Test Switch

### PR DESCRIPTION
**Pull request by JamieB-gu**
It was added in #25357 but I can't see any evidence that it was ever used.

The original description suggests that it would have been used in DCR, but looking through DCR PRs and the history of the DCR `ab-tests` file I can't ever see corresponding code being added: https://github.com/guardian/dotcom-rendering/blob/6763a40e8051c156b8ae5339d25a44f200357009/dotcom-rendering/src/experiments/ab-tests.ts

When it was extended in #26219 there was a suggestion that it wasn't in use yet but would be in future. It was also extended a couple of other times in #26066 and #26583.

However, when experiments were implemented by the Editorial Experience team, in #26189 and #26418, it looks like other switches were used? For both of these I *can* see corresponding changes in DCR:

- https://github.com/guardian/dotcom-rendering/pull/7829
- https://github.com/guardian/dotcom-rendering/pull/8439

In conclusion, it appears that it was originally intended for use with these deeply read tests, but when it came time to implement them other switches were used instead. Therefore, rather than extending to prevent it expiring, it seems it can be safely deleted.

Does this seem correct or have I missed something @marjisound @abeddow91 @sophie-macmillan ?

### CODE REVIEW

**Review by marjisound — APPROVED** [review]
@JamieB-gu Thanks Jamie for looking at this. Yes when I was working on the deeply read I also noticed this server side AB test and wasn't sure why it was there and if it was for something other than the deeply read component we were implementing. 

You are right that this AB test was not needed and can be deleted as we have experiment type AB test set up in DCR

**Review by mxdvl — APPROVED** [review]
We might revisit this in the future, but it is not currently needed!
